travvy/src/components/AttractionsList.vue

```diff
@@ -1,14 +1,126 @@
 <template>
-  <div>
-      <h1>Attractions based on your preferences</h1>
+  <div class="AttractionsList">
+      <!-- this adds the profile button, in later sprints it will take the user to their profile page-->
+      <input type="button" value="Profile" v-on:click="reverseMessage" style="float: right;" class="profile"> 
+
+      <!-- include the toronto heading, an image, and some info about Toronto -->
+      <br><br><br>
+      <h1>Toronto, Canada</h1>
+      <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
+      <br>
+      <h3>Description</h3>
+      <p>Toronto is the largest city in Canada, is the capital of Ontario, and is home to attractions such as the CN tower.</p>
+      <h3>COVID-19</h3>
+      <p>Active cases: 2684</p>
+      <p>Trend: Numbers increasing</p>
+      <h3>Safety Guidelines</h3>
+      <p>Toronto is in phase 2</p>
```

Comment:
maybe in future sprints we can describe what "phase 2" means for people from outside